contracts/ARC20.sol

```diff
@@ -0,0 +1,219 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.7.0;
```

Comment:
you probably want to upper bound this as < 0.8.0

Reply:
Agreed—per hardhart

```zsh
Solidity 0.8.0 is not fully supported yet. You can still use Hardhat, but some features, like stack traces, might not work correctly.
```